ziti/src/test/kotlin/org/openziti/IdentityConfigTest.kt

```diff
@@ -0,0 +1,38 @@
+		   "key":"-----BEGIN PRIVATE KEY-----\nMIGHAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBG0wawIBAQQgyh2aOTEM6t3G7Zh5\nO280GnMu/b7nvnoHOhaDMCKPyDehRANCAAR0DtFWsxJVi6yJxcH8OAHsKp3WLoTT\nnwhTZcYTjVARxYKZMGWsziVAAuuZPb1NV4skL1LvITGQgK1wAjUwU4s5\n-----END PRIVATE KEY-----\n",
```

Comment:
i'm shocked github didn't flag this :)

Reply:
I think (or hope) that codeql recognizes test code